lib/internal/test_runner/coverage.js

```diff
@@ -321,6 +341,10 @@ function mapRangeToLines(range, lines) {
             endOffset >= line.endOffset) {
           line.covered = false;
         }
+        if (count > 0 && startOffset <= line.startOffset &&
+            endOffset >= line.endOffset) {
```

Comment:
nit:
```suggestion
        if (
          count > 0
          && startOffset <= line.startOffset
          && endOffset >= line.endOffset
        ) {
```

Reply:
This matches the existing style, in particular the conditional directly above it. I've tried to stay away from changing formatting too much.